src/components/BurgerMenu/BurgerMenu.module.scss

```diff
@@ -0,0 +1,136 @@
+@use "../../styles/utils/vars" as *;
+@use "../../styles/fonts/typography.scss" as *;
+@use "../../styles/utils/mixins" as *;
+
+$bottom-height: 64px;
+$top-height: 48px;
+$padding-top: 16px;
+$burger-top-margin_bottom: 32px;
+$burger-item-margin: 24px;
+$burger-icon-padding: 22px 72px;
+
+// waiting for header
+/* .burger {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+
+  transform: translateX(-100%);
+  transition: transform 0.5s;
+}
+
+.burger:target {
+  transform: translate(0);
+} */
+
+.top_actions {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.burger__content {
+  height: 100vh;
+  padding: 0;
+  margin: 0;
+}
+
+.burger__top {
+  height: $top-height;
+  padding-left: $padding-top;
+  margin-bottom: $burger-top-margin_bottom;
+
+  border-bottom: 1px solid $secondary-color;
+}
+
+.burger_top_actions {
+  @extend .burger__top;
+  @extend .top_actions;
+}
+
+.logo {
+  &__img {
+    @include onMobile {
+      height: 20.8px;
+      width: 64px;
+    }
+  }
+}
+
+.burger__top_close {
+  display: block;
+  height: $top-height;
+  padding: $padding-top;
+
+  border-left: 1px solid $secondary-color;
+}
+
+.burger__list {
+  display: block;
+  padding: 0;
+
+  text-align: center;
+  list-style: none;
+}
+
+.burger__item {
+  margin-bottom: $burger-item-margin;
+}
+
+.burger__link {
```

Comment:
![telegram-cloud-photo-size-2-5195406107730889569-x](https://github.com/fe-feb23-code-hunters/fe_phone-catalog/assets/107584278/9647f194-1a20-4570-ba1e-09341d8717e2)
Yulia learned how to implement bem to the module scss classes, here is an example of how to import them. Maybe we can use it somewhere in order to write less code?